api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -319,6 +319,29 @@ message AssistCompletionEvent {
   int64 completion_tokens = 4;
 }
 
+// AssistExecutionEvent is an event that is emitted when an Assist command
+// execution is triggered by the user.
+message AssistExecutionEvent {
+  // ConversationId is the UUID that identifies a single Assist conversation
+  string conversation_id = 1;
+  // NodeCount is the number of nodes the command was executed on
+  int64 node_count = 2;
+  // TotalTokens is the total amount of token used to satisfy this request
+  int64 total_tokens = 3;
+  // PromptTokens is the amount of estimated tokens used by the prompt
+  int64 prompt_tokens = 4;
+  // CompletionTokens is the amount of tokens that the completion response consists of
+  int64 completion_tokens = 5;
+}
+
+// AssistNewConversationEvent is an event that is emitted for each new Assist
```

Comment:
Are we classifying individual messages (requests) by the user, or the entire conversation? In the first case, I don't think naming the message "new conversation" makes sense.

Reply:
Today we're classifying the first message of a conversation. The main goal is to measure why the user is invoking assist. I assumed each discussion had a single goal to make things simpler. In the future, if we improve the classification and take all messages into account, I think it's safe to say we'll want to send one category for the whole conversation.